peek.yml

```diff
@@ -0,0 +1,5 @@
+version: 2
+main:
+  type: static
+  path: /mianderson1.github.io
```

Comment:
Since your source and build directory are the same, you can replace this line with `path: .`